extensions/ql-vscode/src/extension.ts

```diff
@@ -871,6 +939,25 @@ async function activateWithInstalledDistribution(
       queryServerLogger,
     ),
   );
+
+  // Since we are tracking extension usage through commands, this command mirrors the runQuery command
+  ctx.subscriptions.push(
+    commandRunnerWithProgress(
+      "codeQL.runQueryContextEditor",
+      async (
+        progress: ProgressCallback,
+        token: CancellationToken,
+        uri: Uri | undefined,
+      ) => await compileAndRunQuery(false, uri, progress, token, undefined),
```

Comment:
```suggestion
      compileAndRunQueryOnMultipleDatabases,
```
This can be shortened to just pass in the reference to the method, instead of defining a new anonymous function with the same parameters. Should be just as type-safe but with less repetition.

Reply:
Talked about this in person with @norascheuch, but I was totally wrong with this (and then next comment too), since here I'm suggesting replacing `compileAndRunQuery` with `compileAndRunQueryOnMultipleDatabases` 🤦 . And we can ignore the bit about using a reference to the method too.